taxonomy-editor-frontend/src/pages/editentry/index.tsx

```diff
@@ -85,7 +91,7 @@ const EditEntry = ({ addNavLinks }) => {
           </DialogContentText>
         </DialogContent>
         <DialogActions>
-          <Button onClick={handleCloseDeleteDialog}>Cancel</Button>
+          <Button onClick={() => setOpenDeleteDialog(false)}>Cancel</Button>
           <Button
             sx={{ color: "#ff0000" }}
```

Comment:
Please add this hex in the constants, and accordingly change wherever this color is used.

Reply:
now using `color` prop